quickstarts/lighttpd/config.yml

```diff
@@ -0,0 +1,39 @@
+
+slug: lighttpd
+description: |
+  ## Comprehensive monitoring for your lighttpd server
+
+  lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure, and adaptable.
```

Comment:
lighttpd should still be capitalized when it's the start of a sentence.

Reply:
okay @paperclypse . I applied the above changes. 
If possible can you please once check [lighttpd page](https://www.lighttpd.net/), I found there it started with small letter in the starting of a sentence as well.

Reply:
For the start of a sentence, I'd prefer to override their style choice (lowercase at start of sentence) with ours (always capitalizing the start of a sentence). I looked through their docs and they're not consistent on this point, so I feel pretty okay with capitalizing it at the start of a sentence.

Reply:
Okay @paperclypse, no problem. I have applied all the changes and pushed. Kindly review and let me know if any more suggestions or changes required.
Thank you.

Reply:
Looks good to me, thanks!